samples/react-pnp-js-sample/package.json

```
@@ -9,30 +9,31 @@
     "test": "gulp test"
   },
   "dependencies": {
-    "@microsoft/sp-core-library": "1.15.2",
-    "@microsoft/sp-lodash-subset": "1.15.2",
-    "@microsoft/sp-office-ui-fabric-core": "1.15.2",
-    "@microsoft/sp-property-pane": "1.15.2",
-    "@microsoft/sp-webpart-base": "1.15.2",
-    "@pnp/logging": "^3.6.0",
-    "@pnp/sp": "^3.6.0",
+    "@microsoft/generator-sharepoint": "^1.20.0",
+    "@microsoft/sp-core-library": "^1.20.0",
+    "@microsoft/sp-lodash-subset": "^1.20.0",
+    "@microsoft/sp-office-ui-fabric-core": "^1.20.0",
+    "@microsoft/sp-property-pane": "^1.20.0",
+    "@microsoft/sp-webpart-base": "^1.20.0",
+    "@pnp/logging": "^3.26.0",
+    "@pnp/sp": "^4.6.0",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "tslib": "2.3.1"
   },
   "devDependencies": {
+    "@microsoft/eslint-config-spfx": "^1.20.0",
+    "@microsoft/eslint-plugin-spfx": "^1.20.0",
     "@microsoft/rush-stack-compiler-4.5": "^0.2.2",
     "@microsoft/sp-build-web": "1.20.1",
```

This should align with SPFx version
```suggestion
    "@microsoft/sp-build-web": "1.20.0",
```

---

@Adam-it solution tested using "@microsoft/sp-build-web": "1.20.1" and build fails.